### PR TITLE
Test round

### PR DESCRIPTION
**Pull request by JulienBenoit7**

### CODE REVIEW

**Comment by rocambille** [comment]
conflicts to resolve

File: frontend/src/components/board-components/Rounds.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+
+function Rounds() {
+  const rounds = ["SRM", "IBU", "ABV", "EBC"];
+  const [value, setValue] = useState();
+
+  const ChangeRound = (arr) => {
```
Comment (rocambille) [inline frontend/src/components/board-components/Rounds.jsx]: changeRound, not ChangeRound

File: frontend/src/App.css [hunk 2]
```diff
@@ -241,8 +241,9 @@ nav img {
 .scoreBoard-name {
   padding: 1rem;
   text-align: center;
-  background-color: #ad1a1ac1;
+  background-color: transparent;
   width: 8rem;
+
```
Comment (rocambille) [inline frontend/src/App.css]: remove this empty line

File: frontend/src/components/board-components/Rounds.jsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    const selection = Math.round(Math.random() * (arr.length - 1));
+    setValue(arr[selection]);
+  };
+
+  return (
```
Comment (rocambille) [inline frontend/src/components/board-components/Rounds.jsx]: looks like a single round, so call it Round

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/src/components/board-components/Rounds.jsx (hunk 3 above)
Comment (Justine-Merlin) [reply frontend/src/components/board-components/Rounds.jsx]: This is not resolved. The goal is to rename your component `Rounds` in `Round` :eyes:

**Review by Justine-Merlin — APPROVED** [review]

**Pull request merged**